plugins/woocommerce-admin/client/layout/footer/footer.scss

```diff
@@ -1,23 +1,28 @@
-.woocommerce-layout__footer {
-	background: $studio-white;
-	border-top: 1px solid $gray-200;
-	box-sizing: border-box;
-	padding: 0;
-	position: fixed;
-	width: calc(100% - 160px);
-	bottom: -1px; /* to hide the border when no visible children */
-	z-index: 1001; /* on top of #wp-content-editor-tools */
+.woocommerce-admin-page__add-product {
```

Comment:
Is there a reason you decided to wrap the entire footer css with `woocommerce-admin-page__add-product`?
This means it will break on any other WooCommerce related page, also this breaks on the `edit` product page.

I would say we should leave it as is and remove the wrapper again, unless you have a strong reason to do otherwise?

Reply:
Good catch! I fixed this in the commit [ee1e52c](https://github.com/woocommerce/woocommerce/pull/38941/commits/ee1e52c573e5225ad12041d1af38f2956cc0ce7e).